src/components/Operation/Operation.tsx

```diff
@@ -363,28 +395,37 @@ export class Operation extends React.Component<OperationProps, OperationState> {
                 <Description>
                   {description !== undefined && <Markdown source={description} />}
                   {externalDocs && <ExternalDocumentation externalDocs={externalDocs} />}
+                  
                 </Description>
+                
               )}
-              <Extensions extensions={operation.extensions} />
-              <SecurityRequirements securities={operation.security} />
+                            <Extensions extensions={operation.extensions} />
+                            <SecurityRequirements securities={operation.security} />
+              <H2>
+              {!options.pathInMiddlePanel && !isWebhook && <Endpoint operation={operation} />}
```

Comment:
this block, including `<H2/>` tags, has to go on line 428, right after `<TryOutHeader/>` in order for header to be on the right panel